internal/core/Cargo.toml

```diff
@@ -88,7 +87,7 @@ serde = { version = "1.0.163", features = ["derive"], optional = true }
 gettext-rs = { version = "0.7", optional = true, features = ["gettext-system"] }
 
 [target.'cfg(target_arch = "wasm32")'.dependencies]
-instant = { version = "0.1", features = [ "wasm-bindgen", "now" ] }
+web-time = { version = "0.2", optional = true }
```

Comment:
We used to depend on `instant`  on all platform.  But winit only depends on it for wasm, so i thought i'd do the same.  I guess this reduce the compile time by a negligible amount